js/models.js

```diff
@@ -104,6 +104,31 @@ class StoryList {
       /*
     old code, might be useful at some point */
   }
+
+  /* Recieves a user obj and a storyId
+  *   -verifies story belongs to user
+  *   -deletes story from api
+  */
+  async deleteStoryFromList(user, storyId){
+    const belongsToUser = user.ownStories.some((story) => (story.storyId == storyId));
+    if(belongsToUser){
+      try{
+        await axios({
+          url: `${BASE_URL}/stories/${storyId}`,
+          method: "DELETE",
+          data: {
+            "token": user.loginToken
+          }
+        });
+        currentUser = await User.updateUser();
+        displayMyStories();
+      }
+      catch (e){
+        console.error("Failed to delete story", e)
```

Comment:
Could return `false` here (or `true` above) to check for it in `deleteStory()` and show feedback to user.